packages/dashboard/src/components/Card/Card.tsx

```diff
@@ -33,6 +33,7 @@ export class Card extends React.Component<ICardProps, ICardState> {
           cardTitle={cardFrameContent.cardTitle}
           cardDropDownOptions={cardFrameContent.cardDropDownOptions}
           href={cardFrameContent.href}
+          draggable={draggable === undefined ? true : draggable}
```

Comment:
I'm guessing this means it's true by default? Typically we try to set props to be the exception case, so disableDrag is typically false, but can be set to try via <Foo disableDrag />

https://github.com/OfficeDev/office-ui-fabric-react/wiki/Component-Design#naming-guidance